platforms/win32/plugins/CroquetPlugin/sqWin32CroquetPlugin.c

```diff
@@ -8,7 +8,7 @@ static BOOLEAN (__stdcall *RtlGenRandom)(PVOID, ULONG) = NULL;
 int ioGatherEntropy(char *bufPtr, int bufSize) {
   if(!loaded) {
     loaded = 1;
-    hAdvApi32 = LoadLibrary("advapi32.dll");
+    hAdvApi32 = LoadLibraryA("advapi32.dll");

```

Comment:
What about

```suggestion
    hAdvApi32 = LoadLibrary(TEXT("advapi32.dll"));
```

instead?